src/include/rbd/librbd.h

```diff
@@ -144,6 +144,8 @@ enum {
   RBD_IMAGE_OPTION_JOURNAL_ORDER = 5,
   RBD_IMAGE_OPTION_JOURNAL_SPLAY_WIDTH = 6,
   RBD_IMAGE_OPTION_JOURNAL_POOL = 7,
+  RBD_IMAGE_OPTION_FEATURES_SET = 8,
```

Comment:
Example:
There is an image with no-object-map. But I want to clone a new one with object-map feature enabled. Then I need to not set RBD_IMAGE_OPTION_FEATURES to tell librbd to inherit features from parent and set RBD_IMAGE_OPTION_FEATURES_SET to apply object-map bit in features. 

RBD_IMAGE_OPTION_FEATURES_[SET|CLEAR] are always applied no matter RBD_IMAGE_OPTION_FEATURES specified or not. And I am kicking out the bits which are both specified in SET and CLEAR. 

This is the main logic of it:
https://github.com/ceph/ceph/pull/9334/files#diff-254555dde8dcfb7fb908791ab8214b92R1269